### ISSUE TITLE
Failure using example in README.md

### ISSUE THREAD

**Issue by Brett-Best** [fumito-ito/PrivacyManifest#1]
Getting the below error when trying to use the example in the README.md.
I tried a bunch of different URLs but couldn't get it to work, any ideas?
Thanks!
```
x pkl eval PrivacyInfo.pkl > PrivacyInfo.xcprivacy
–– Pkl Error ––
Received unexpected status code `404` when making request `GET https://pkg.pkl-lang.org/github.com/fumito-ito/PrivacyManifest/PrivacyManifest@0.0.2`.

1 | amends "package://pkg.pkl-lang.org/github.com/fumito-ito/PrivacyManifest/PrivacyManifest@0.0.2#/PrivacyManifestFile.pkl"
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
at PrivacyInfo (file:///Users/brettbest/Developer/DummyApp/PrivacyInfo.pkl, line 1)
```

**Comment by fumito-ito** [comment]
hi @Brett-Best, thanks for you reporting ! I fixed wrong release and tag name on [0.1.0](https://github.com/fumito-ito/PrivacyManifest/releases/tag/PrivacyManifest%400.1.0). Please use it with following scripts.

```pkl
amends "package://pkg.pkl-lang.org/github.com/fumito-ito/PrivacyManifest/PrivacyManifest@0.1.0#/PrivacyManifestFile.pkl"

NSPrivacyTracking = true
NSPrivacyTrackingDomains {
    "com.github.your.tracking.domain"
}
```

**Comment by Brett-Best** [comment]
Thanks, that worked @fumito-ito!
